packages/react/src/components/slider/Slider.tsx

```diff
@@ -1,4 +1,4 @@
-import 'rc-slider/assets/index.css';
+// import 'rc-slider/assets/index.css';
```

Comment:
all those imports in comment, it's forgotten or Next is working that way? :eyes:

Reply:
Oops, this one I forgot to fix